src/app/hooks/useTransactions.ts

```diff
@@ -16,13 +16,14 @@ export const useTransactions = () => {
         const getTransactionsResponse = await api.getTransactions({
           limit,
         });
-
         const transactions = getTransactionsResponse.transactions.map(
           (transaction) => ({
             ...transaction,
             title: transaction.memo,
-            date: dayjs(transaction.settleDate).fromNow(),
-            timestamp: transaction.settleDate,
+            date: dayjs(
```

Comment:
is it possible to rename this `date` field? to me it is not a date

Reply:
how about "timeAgo"  or "relativeDateTime" ?

Reply:
changed to timeAgo